### PR TITLE
Document subctl uninstall command

### LINKED ISSUE #684: Document the new `subctl uninstall` command
- [x] Document the new uninstall command
- [x] Consider whether we should remove the manual uninstall guide completely (https://submariner.io/operations/cleanup/)  

Depends on https://github.com/submariner-io/submariner-operator/pull/1877

### PR DESCRIPTION
**Pull request by sridhargaddam**
Fixes: https://github.com/submariner-io/submariner-website/issues/684
Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>

### CODE REVIEW

File: src/content/operations/deployment/subctl/_index.en.md [hunk 1]
```diff
@@ -454,3 +454,27 @@ This command removes the labels from gateway nodes after Submariner uninstallati
...
+* Delete the broker namespace, if present.
+* Unlabel the gateway nodes.
+
+#### `uninstall` flags
+
+| Flag                             | Description
+|:---------------------------------|:--------------------------------------------------------------------------------------------------------------------------|
+| `--kubeconfig` `<string>`        | Absolute path(s) to the kubeconfig file(s)
+| `--namespace` `<string>`         | namespace in which Submariner is installed (default "submariner-operator")
+| `--yes`                          | automatically answer yes to confirmation prompt
```
Comment (dfarrell07) [inline src/content/operations/deployment/subctl/_index.en.md]:
```suggestion
| `--yes`                          | Automatically answer yes to confirmation prompt
```

File: src/content/operations/deployment/subctl/_index.en.md [hunk 2]
```diff
@@ -454,3 +454,27 @@ This command removes the labels from gateway nodes after Submariner uninstallati
...
+* Delete the Submariner namespace.
+* Delete the broker namespace, if present.
+* Unlabel the gateway nodes.
+
+#### `uninstall` flags
+
+| Flag                             | Description
+|:---------------------------------|:--------------------------------------------------------------------------------------------------------------------------|
+| `--kubeconfig` `<string>`        | Absolute path(s) to the kubeconfig file(s)
+| `--namespace` `<string>`         | namespace in which Submariner is installed (default "submariner-operator")
```
Comment (dfarrell07) [inline src/content/operations/deployment/subctl/_index.en.md]:
```suggestion
| `--namespace` `<string>`         | Namespace in which Submariner is installed (default "submariner-operator")
```

**Review by dfarrell07 — APPROVED** [review]
A few minor suggestions, but non-blocking for me.

File: src/content/operations/deployment/subctl/_index.en.md [hunk 3]
```diff
@@ -454,3 +454,27 @@ This command removes the labels from gateway nodes after Submariner uninstallati
...
+This command uninstalls Submariner and its components.
+
+The following steps are performed:
+
+* Delete Submariner ClusterRoles and ClusterRoleBindings.
+* Delete the submariner.io CRDs.
+* Delete the iptables/routes/ipsets programmed on the nodes.
+* Delete the tunnel interfaces created for internal/external communication.
+* Delete the Submariner namespace.
+* Delete the broker namespace, if present.
```
Comment (nyechiel) [inline src/content/operations/deployment/subctl/_index.en.md]:
```suggestion
* Delete the Broker namespace, if present.
```

File: src/content/operations/deployment/subctl/_index.en.md [hunk 4]
```diff
@@ -454,3 +454,27 @@ This command removes the labels from gateway nodes after Submariner uninstallati
...
+
+`subctl uninstall [flags]`
+
+This command uninstalls Submariner and its components.
+
+The following steps are performed:
+
+* Delete Submariner ClusterRoles and ClusterRoleBindings.
+* Delete the submariner.io CRDs.
+* Delete the iptables/routes/ipsets programmed on the nodes.
```
Comment (nyechiel) [inline src/content/operations/deployment/subctl/_index.en.md]:
```suggestion
* Delete the routing entries (iptables/routes/ipsets) programmed on the nodes.
```

**Comment by sridhargaddam** [comment]
Thanks for reviewing @nyechiel @dfarrell07

**Review by nyechiel — APPROVED** [review]

**Pull request merged**